src/app/additionalquestionsform/AdditionalQuestionsForm.tsx

```diff
@@ -0,0 +1,172 @@
+  ) => {
+    const { name } = event.target;
+    setErrors({ ...errors, [name]: "" });
+  };
+
+  const handleChange = (
+    event: ChangeEvent<HTMLInputElement | HTMLTextAreaElement>,
+  ) => {
+    const { name, value } = event.target;
+    setFormData({ ...formData, [name]: value });
+  };
+
+  // eslint-disable-next-line unicorn/consistent-function-scoping
+  const validateEmail = (email: string) => {
+    const emailCheck = /^[\w%+.-]+@[\d.A-Za-z-]+\.[A-Za-z]{2,}$/;
+    return emailCheck.test(email);
+  };
+
+  const validateForm = () => {
+    const errors: { [key: string]: string } = {};
+
+    const { name, email, message } = formData;
+
+    if (!name) errors.name = "Name is required";
+    if (!email) {
+      errors.email = "Email is required";
+    } else if (!validateEmail(email)) {
+      errors.email = "Email is invalid";
+    }
+    if (!message) errors.message = "Message is required";
+    return errors;
+  };
+
+  const handleSubmit = async (event: FormEvent) => {
+    event.preventDefault();
+    const validationErrors = validateForm();
+    if (Object.keys(validationErrors).length > 0) {
+      setErrors(validationErrors);
+    } else {
+      try {
+        const response = await fetch("/api/submit-question", {
+          method: "POST",
+          headers: {
+            "Content-Type": "application/json",
+          },
+          body: JSON.stringify(formData),
+        });
+        if (response.ok) {
+          setSuccessMessage("Your question has been submitted successfully.");
+          setErrorMessage("");
+          setFormData({ name: "", email: "", message: "" });
+        } else {
+          throw new Error("Network response was not ok");
+        }
+      } catch (error) {
+        setErrorMessage(
+          "There was an error submitting your question. Please try again.",
+        );
+        setSuccessMessage("");
+        setFormData({ name: "", email: "", message: "" });
+        console.log(error);
+      }
+    }
+  };
+
+  return (
+    <div className="mb-[11.375rem] mt-[2.75rem] flex w-full max-w-[676px] flex-col gap-[28px] md:mb-[4rem] md:gap-[4.125rem]">
+      <div className="flex flex-col items-center gap-[.9375rem] text-center md:gap-[9px]">
+        <h3 className="text-[1.25rem] font-[700] leading-[33.89px] text-[#F97316] md:text-[1.75rem]">
+          Still have questions?
+        </h3>
+        <p className="text-[1rem] font-[400] leading-[1.3613rem] text-[#525252] md:text-[1.125rem]">
+          Fill the form and enter your message
+        </p>
+      </div>
+      <form
+        onSubmit={handleSubmit}
+        className="flex w-full flex-col gap-[27px] md:gap-[1.5rem]"
+        noValidate
+      >
+        <div className="flex flex-col gap-[8px] md:gap-[.75rem]">
+          <label
+            htmlFor="email"
+            className="text-[1rem] font-[400] leading-[1.5rem] text-[#434343] md:text-[1.25rem]"
+          >
+            Email:
+          </label>
+          <input
```

Comment:
Please use the reusable input component installed on the repo already
Also, I believe there's a package called zod that can help you with form validation. You can check the shadcn docs to understand how to make use of it